src/SearchForm/SearchForm.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react"
+import GenreSelect from "../GenreSelect/GenreSelect";
+
+function SearchForm(props) {
+  const Genres = ['Action', 'Comedy', 'Drama', 'Fantasy']
+  const [searchText, setSearchText] = useState('');
+
+  return <>
+
+    <nav className="navbar navbar-expand-lg navbar-light bg-light">
+      <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
let's get rid of the attributes you don't need for now, let's avoid extra code in general

Reply:
fixed